api.planx.uk/helpers.ts

```diff
@@ -418,6 +420,7 @@ const dataMerged = async (
         data: {
           text: `${team.slug}/${slug}`,
           flattenedFromExternalPortal: true,
+          templatedFrom: response.templatedFrom,
```

Comment:
When flattening flows, all 'external portals'/'nested flows' get re-categorised as 'internal portals'/'folders'. 

We already have a prop `flattenedFromExternalPortal` which lets us quickly determine it's origin, and we can similar add another prop `templatedFrom` to distinguish if the 'external portal'/'nested flow' is a templated flow itself.